extensions/transmission/src/utils/renderCells.tsx

```diff
@@ -54,12 +54,18 @@ export async function renderPieces({
 
   const cellsCount = Math.pow(18, 2);
 
-  const bits: boolean[] = [];
+  let bits: boolean[] = [];
   bitfield.forEach((bit) => bits.push(bit));
 
+  // if we don't have enough pieces to compose a graph, we multiply them
+  if (bits.length < cellsCount) {
+    const diff = Math.ceil(cellsCount / bits.length);
+    bits = bits.map((bit) => Array.from({ length: diff }, () => bit)).flat();
```

Comment:
basically we go from `[true, false false]` to `[true, true, true, false, false, false, false, false, false]` so that we have at least 8^2 elements.